cmd/earthly/build_cmd.go

```diff
@@ -179,13 +179,20 @@ func (app *earthlyApp) actionBuildImp(cliCtx *cli.Context, flagArgs, nonFlagArgs
 		return err
 	}
 
+	earthfileOrgName := "my-org" // TODO (vladaionescu): Detect this.
+	earthfileProjectName := ""   // TODO (vladaionescu): Detect this.
+	app.logbusSetup.SetOrgAndProject(earthfileOrgName, earthfileProjectName)
+
 	// Default upload logs, unless explicitly configured
 	if !app.cfg.Global.DisableLogSharing {
 		if cloudClient.IsLoggedIn(cliCtx.Context) {
 			if app.logstreamUpload {
-				earthfileOrgName := "my-org" // TODO (vladaionescu): Detect this.
-				earthfileProjectName := ""   // TODO (vladaionescu): Detect this.
-				app.logbusSetup.StartLogStreamer(cliCtx.Context, cloudClient, earthfileOrgName, earthfileProjectName)
+				app.logbusSetup.StartLogStreamer(cliCtx.Context, cloudClient)
+				urlStr := fmt.Sprintf("https://ci.earthly.dev/todourl/%s", app.logbusSetup.InitialManifest.GetBuildId())
```

Comment:
What's a correct URL for this BTW?

Reply:
I don't think it's finalized yet. @mirho or @r-LaForge may have a better idea.

Reply:
That's ok - we can leave it like this for now then

Reply:
I think the path is `/{orgId}/projects/{projectId}/{pathHash}/{pipelineId}/{buildId}`

I'm not entirely sure how to determine the pathHash yet.

Reply:
Perhaps the server should return this URL early in the logstream lifecycle somehow

Reply:
Added a `linear.app` ticket to keep track of this: https://linear.app/earthly-technologies/issue/ET-383/expose-correct-logstream-url-through-earthly-core